Frontend/logindeco.css

```diff
@@ -0,0 +1,80 @@
+img
```

Comment:
try to avoid 'tag' css selector. this might be a problem if you use multiple img tags